wordpress-mobile/WordPress-Android
# Remove unnecessary title interface from basic fragments

We no longer have the title of the first fragment set based on the current site title so I'm removing an unnecessary interface that set the title of toolbars in all the fragments. All the toolbars are inside the fragments so it's no longer needed to set them from the parent activity. 

To test:
- Open the app
- Check that all the main fragments and the Me fragment have correct toolbar titles

PR submission checklist:
